client/src/entrypoints/admin/userbar.js

```diff
@@ -77,9 +77,9 @@ document.addEventListener('DOMContentLoaded', () => {
   function toggleUserbar(e2) {
     e2.stopPropagation();
     if (userbar.classList.contains(isActiveClass)) {
-      hideUserbar();
+      hideUserbar(true);
```

Comment:
This will not really do anything, the function `hideUserbar` does not accept any param.

Reply:
Ok, I will come up with a new approach then.